micrometer-observation-test/src/main/java/io/micrometer/observation/tck/ObservationRegistryCompatibilityKit.java

```diff
@@ -1004,15 +1004,13 @@ public boolean supportsContext(Observation.Context context) {
 
     static class TestObservationConventionWithNameOverrides implements GlobalObservationConvention<TestContext> {
 
-        @Nullable
         @Override
-        public String getName() {
+        public @Nullable String getName() {
             return "conventionOverriddenName";
```

Comment:
I wonder if we shouldn't remove the `@Nullable` here. I guess it complains about overriding a `@Nullable` method then? We actually know this override is not nullable, although I doubt it matters either way for this class.

Reply:
Did it in the next PR (docs + formatting).